nullaway/build.gradle

```diff
@@ -60,6 +60,7 @@ dependencies {
     testCompile deps.test.commonsLang
     testCompile deps.test.commonsLang3
     testCompile project(":test-library-models")
+    testCompile deps.test.lombok
```

Comment:
So I guess Lombok will run on all our test inputs?  I guess this shouldn't cause a problem as long as they don't use Lombok.  But maybe it's sufficient to just compile `test-java-lib-lombok` and not also have a Lombok unit test?

Reply:
Lombok will not run on any tests, this merely adds the library dependency, we are not adding the annotation processor itself. So, the unit test exclusively tests the defaults for `@Builder` but nothing about `@lombok.Generated` or the processor itself.